app/javascript/Pages/Login.jsx

```diff
@@ -1,64 +1,79 @@
-import React from "react";
+import React, { Component } from "react";
 import { Inertia } from "@inertiajs/inertia";
-import Layout from "./Layout";
+import Navbar from "./components/Navbar/Navbar";
 import { Head, Link } from "@inertiajs/inertia-react";
+import "../stylesheets/body.css";
+import "../stylesheets/login.css";
 
-export default function Login(props) {
-  const ErrorMessage = () => {
-    if (props.error) {
-      return <p>{props.error}</p>;
-    }
-    return null;
-  };
+class Login extends Component {
+  render() {
+    const ErrorMessage = () => {
+      if (this.props.error) {
+        return <p>{this.props.error}</p>;
+      }
+      return null;
+    };
 
-  const handleSubmit = (e) => {
-    const token = document.querySelector(
-      "[name=csrf-token]"
-    ).content;
+    const handleSubmit = (e) => {
+      const token = document.querySelector("[name=csrf-token]").content;
 
-    const data = {
-      username: username.value,
-      password: password.value,
-    };
+      const data = {
+        username: username.value,
+        password: password.value,
+      };
 
-    e.preventDefault();
-    Inertia.post("/login", data, {
-      headers: {
-        "X-CSRF-Token": token,
-      },
-    });
-  };
+      e.preventDefault();
+      Inertia.post("/login", data, {
+        headers: {
+          "X-CSRF-Token": token,
+        },
+      });
+    };
 
-  return (
-    <main>
-      <Head title="Welcome" />
-      <ErrorMessage />
-      <h1>Login</h1>
-      <form onSubmit={handleSubmit}>
-        <label htmlFor="username">Username/Email:</label>
-        <br />
-        <input
-          type="text"
-          id="username"
-          required
-          name="username"
+    return (
+      <main>
+        <Navbar />
+        <img
+          src={require("../images/scene_1.svg")}
+          alt="logo"
+          className="scene-1-login"
         />
-        <br />
-        <label htmlFor="password">Password:</label>
-        <br />
-        <input
-          type="password"
-          required
-          id="password"
-          name="password"
-        />
-        <br />
+        <div className="login_content">
+          <Head title="Welcome" />
+          <ErrorMessage />
+          <h1>Log in to your account</h1>
+          <form onSubmit={handleSubmit}>
+            <label htmlFor="username"></label>
```

Comment:
**question (blocking)**: why are these labels empty? if you are relying on the placeholder of the input for the label it's a common practice to place the input inside the label. Also, add the placeholder as an aria-label in the label so that screen-readers can see it.